pages/common/retry.md

```diff
@@ -0,0 +1,16 @@
+# retry
+
+> Repeat command until a criteria is met, usually success.
```

Comment:
```suggestion
> Repeat a command until the command succeeds when a criterion is met.
```

Reply:
criterion?


```suggestion
> Repeat a command until a criteria is met.
```

Reply:
> criterion?

Yep, since it's a noun.

Reply:
Never seen the usage of the word.

Reply:
I'm impartial but if we want complete formality rather than whats spoken mostly in every day English then criterion is correct.